data.tf

```diff
@@ -18,6 +18,6 @@ data "aws_eks_cluster_auth" "this" {
 
 # To Authenticate with ECR Public in eu-east-1
 data "aws_ecrpublic_authorization_token" "token" {
-  provider = aws.oregon
+  provider = aws.virginia
```

Comment:
Is the region correct?

Reply:
Yes, I changed it to Virginia because I got the below error:
Error: getting ECR Public authorization token: UnsupportedCommandException: GetAuthorizationToken command is only supported in us-east-1.
│ 
│   with data.aws_ecrpublic_authorization_token.token,
│   on data.tf line 20, in data "aws_ecrpublic_authorization_token" "token":
│   20: data "aws_ecrpublic_authorization_token" "token" {